src/main/java/org/fundacionjala/virtualassistant/repository/ASRClient.java

```diff
@@ -0,0 +1,13 @@
+package org.fundacionjala.virtualassistant.repository;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import javax.sound.sampled.UnsupportedAudioFileException;
+
+public interface ASRClient {
+
+    void convertToText(Path audioFile) throws IOException, UnsupportedAudioFileException;
```

Comment:
avoid having void methods? what are you expecting this method to do? if you want to convert an AudioFile to a text you will expect to have a text